### PR TITLE
Packaging healthcheck

### PR DESCRIPTION
**Pull request by kinvaris**
in reference to https://github.com/openvstorage/openvstorage-health-check/issues/71 & https://github.com/openvstorage/integrationtests/issues/274

### CODE REVIEW

**Comment by kinvaris** [comment]
@pploegaert & @khenderick can you please review?

File: bin/post-install.sh [hunk 1]
```diff
@@ -22,5 +22,4 @@ chmod 755 /usr/bin/ovs
 wget https://bootstrap.pypa.io/get-pip.py; python get-pip.py
 #pip install flower
 pip install psutil
-pip install xmltodict
 pip install timeout-decorator
```
Comment (pploegaert) [inline bin/post-install.sh]: newline missing ...

File: packaging/debian/debian/openvstorage-health-check.dirs [hunk 2]
```diff
@@ -0,0 +1 @@
+opt/OpenvStorage/config/healthcheck
```
Comment (pploegaert) [inline packaging/debian/debian/openvstorage-health-check.dirs]: newline missing ...

File: packaging/__init__.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Packaging package
+"""
```
Comment (pploegaert) [inline packaging/__init__.py]: incorrect license

File: bin/post-install.sh (hunk 1 above)
Comment (kinvaris) [inline bin/post-install.sh]: these are obsolete, have to be removed

File: packaging/debian/debian/openvstorage-health-check.dirs (hunk 2 above)
Comment (kinvaris) [inline packaging/debian/debian/openvstorage-health-check.dirs]: done

**Comment by khenderick** [comment]
Make sure to do the required changes to https://github.com/openvstorage/framework-tools/blob/master/packaging/settings.json so the packging logic knows about this repo/package.

File: packaging/debian/debian/control [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+Maintainer: Jonas Libbrecht <jonas.libbrecht@openvstorage.com>
+Standards-Version: 3.9.4
+Section: python
+Priority: optional
+Build-Depends: python-all (>= 2.7.2), debhelper (>= 9)
+
+Package: openvstorage-health-check
+Architecture: amd64
+Pre-Depends: python (>= 2.7.2)
+Depends: python-psutil (>= 4.3.0), python-timeout-decorator (>= 0.3.2)
```
Comment (khenderick) [inline packaging/debian/debian/control]: Doesn't it require the ```openvstorage``` package?

File: packaging/debian/debian/copyright [hunk 5]
```diff
@@ -0,0 +1,31 @@
+                  Open vStorage Non-Commercial License
```
Comment (khenderick) [inline packaging/debian/debian/copyright]: A general remark; shouldn't the healthcheck be updated to AGPL-3.0 as the other repos?

File: packaging/settings.json [hunk 6]
```diff
@@ -0,0 +1,10 @@
+{
+    "package_name": "openvstorage-health-check",
+    "product_name": "Open vStorage - Health-Check",
+    "source_contents": "--transform 's,^,{0}-{1}/,' config/healthcheck *.txt",
```
Comment (khenderick) [inline packaging/settings.json]: I think you want to include more in your sources (the directories ```ovs```, ```config``` and probably others as well)

File: packaging/debian/debian/control (hunk 4 above)
Comment (kinvaris) [inline packaging/debian/debian/control]: no we are not including `openvstorage` packages at request of @pploegaert

File: packaging/debian/debian/copyright (hunk 5 above)
Comment (kinvaris) [inline packaging/debian/debian/copyright]: True

**Comment by kinvaris** [comment]
At request of @khenderick: Will remove https://github.com/openvstorage/openvstorage-health-check/blob/packaging/scripts/system/ovs at request and do pull request on framework so we can easily implement plugins/extension scripts in the future via `/opt/OpenvStorage/scripts/`.

So if you would do `ovs healthcheck silent`, the script `/opt/OpenvStorage/scripts/healthcheck.sh` should be available. Further parameters will be passed along the script ...

**Comment by kinvaris** [comment]
@khenderick & @pploegaert, all requested issues in this branch should be resolved/updated.

**Comment by kinvaris** [comment]
approved

**Pull request merged**